tensorflow/core/user_ops/gather_columns.cc

```diff
@@ -0,0 +1,183 @@
+#include "tensorflow/core/framework/op.h"
+//#include "tensorflow/core/framework/shape_inference.h"
```

Comment:
It would be needed if the shape of the output tensor is to be inferred. I left it there as a reminder. Will remove it.